apps/condo/domains/miniapp/schema/B2BAccessToken.js

```diff
@@ -0,0 +1,280 @@
+                    return cookieSignature.sign(sessionId, cookieSecret)
+                }
+                return null
+            },
+        },
+
+        context: {
+            schemaDoc: 'Link to B2BApp and Organization',
+            type: 'Relationship',
+            ref: 'B2BAppContext',
+            isRequired: true,
+            knexOptions: { isNotNullable: true }, // Required relationship only!
+            kmigratorOptions: { null: false, on_delete: 'models.CASCADE' },
+            access: access.updatableOnlyForAdmin,
+        },
+
+        user: {
+            ...SERVICE_USER_FIELD,
+            isRequired: true,
+            access: access.updatableOnlyForAdmin,
+            hooks: {
+                ...SERVICE_USER_FIELD.hooks,
+                async resolveInput ({ operation, existingItem, resolvedData, fieldPath }) {
+                    if (operation === 'create' && !has(resolvedData, fieldPath)) {
+                        const contextId = get(resolvedData, 'context') || get(existingItem, 'context')
+                        const context = await getById('B2BAppContext', contextId)
+                        const [accessRight] = await find('B2BAppAccessRight', {
+                            app: { id: context.app },
+                            deletedAt: null,
+                        })
+                        return accessRight.user
+                    }
+
+                    return existingItem.user
+                },
+            },
+        },
+
+        rightSet: {
+            schemaDoc: 'Right set for token',
+            type: 'Relationship',
+            ref: 'B2BAppAccessRightSet',
+            isRequired: true,
+            knexOptions: { isNotNullable: true }, // Required relationship only!
+            kmigratorOptions: { null: false, on_delete: 'models.CASCADE' },
+            access: access.updatableOnlyForAdmin,
+        },
+
+        expiresAt: {
+            schemaDoc: 'Date, when access by this token will be restricted',
+            type: 'DateTimeUtc',
+            isRequired: false,
+            access: access.updatableOnlyForAdmin,
+            hooks: {
+                validateInput ({ resolvedData, fieldPath, context }) {
+                    const expiresAt = resolvedData[fieldPath]
+                    if (isNil(expiresAt)) {
+                        return
+                    }
+                    if (dayjs(expiresAt).isBefore(dayjs())) {
+                        throw new GQLError(ERRORS.NEGATIVE_TTL, context)
+                    }
+                },
+            },
+        },
+    },
+    hooks: {
+        async validateInput ({ operation, resolvedData, existingItem, context }) {
+            const newItem = {
+                ...(existingItem || {}),
+                ...resolvedData,
+            }
+
+            if (newItem.deletedAt) {
+                return
+            }
+
+            const b2bAppContext = await getByCondition('B2BAppContext', {
```

Comment:
You should check, that all apps in fetched models are the same

Reply:
Already have. 
1. Get B2BAppContext
2. Get B2BAppRightsSet
3. Get B2BAppAccessRight with same app, as b2bAppContext (b2bAppContext.app === accessRightSet.app)
4. Check app equality of context and right set